FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.Common.UnitTests/Extensions/ConfigurationRegistrationExtensionsTests.cs

```diff
@@ -0,0 +1,94 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Health.Fhir.Synapse.Common.Configurations;
+using Microsoft.Health.Fhir.Synapse.Common.Configurations.Arrow;
+using Microsoft.Health.Fhir.Synapse.Common.Exceptions;
+using Microsoft.Health.Fhir.Synapse.Common.Extensions;
+using Microsoft.Health.Fhir.Synapse.Common.Models.Jobs;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Synapse.Common.UnitTests.Extensions
+{
+    public class ConfigurationRegistrationExtensionsTests
```

Comment:
A question: Why we not directly set up unit tests for **ConfigurationValidator**?

Reply:
Because we want to test the configVersion values